dashboard/test/ui/features/acquisition_products/sign_up.feature

```diff
@@ -0,0 +1,22 @@
+@eyes
+Feature: Teacher can create a new account
+
+Scenario: Teacher can create an account with the new school association flow
+
+  Given I am on "http://studio.code.org/users/sign_up"
+  And I fill in the sign up email field with a random email
```

Comment:
And this is the only changed line here

Reply:
Ugh I'm sorry -- should've caught this in review :/